Automattic/pocket-casts-ios
# Hack week: Lock screen scrubber setting

Fixes #2200

Adds a "Lock Screen Scrubbing" setting to the general player to disable (or enable) scrubbing from the lock screen.

<img src="https://github.com/user-attachments/assets/9599620f-39cc-4c91-b04c-06b33c48a0bd" width=300>

## To test

* Play back a podcast and verify changing the playback position in the scrubber still works
* Navigate to Profile > Settings > General and change the "Lock Screen Scrubbing" setting under "Player" from enabled to disabled
* Lock the screen again and verify changing the playback position in the scrubber no longer works

## Checklist
